ONSdigital/rm-sample-service
# Remove rabbit queue and call party service directly

# What and why?
Remove the call to Rabbit to queue the calls to party service, instead directly invoke the party service

# How to test?
Create a new collection exercise and upload a sample, set CE to go live and ensure everything works as expected.

# Trello
https://trello.com/c/KbZGovmD/442-s33-ce-sample-service-calling-party
